src/main/java/lotto/domain/PurchaseCount.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import static lotto.exception.ExceptionMessage.NON_INTEGER_INPUT_FOR_PURCHASE_MONEY;
+import static lotto.exception.ExceptionMessage.NON_MULTIPLE_OF_LOTTO_PRICE_INPUT_FOR_PURCHASE_MONEY;
+import static lotto.util.NumberValidateUtils.isInteger;
+
+public class PurchaseCount {
+
+    private static final int MINIMUM_INPUT = 1000;
+
+    @Getter
+    private final int purchaseCount;
+
+    @Builder
+    public PurchaseCount(String input) {
+        input = input.trim();
+        validate(input);
+
+        this.purchaseCount = Integer.parseInt(input) / Lotto.PRICE;
+    }
+
+    private void validate(String input) {
+        if (!isInteger(input)) {
```

Comment:
`isInteger` 는 직독하면 정수인가? 는 판단문입니다.
다만 내용을 보니 0~9까지 숫자 즉 음이 아닌 정수인지를 정규표현식을 이용해 판단하고 있네요.
isNonNegativeInteger 정도로 더 정확하게 표현해보면 어떨까요?
```suggestion
        if (!isNonNegativeInteger(input)) {
```